test/python/rest/test_rest_bluemix.py

```diff
@@ -15,3 +16,22 @@ def setUp(self):
         Tester.setup_streaming_analytics(self, force_remote_build=True)
         self.sc = StreamingAnalyticsConnection()
         self.test_config[ConfigParams.STREAMS_CONNECTION]=self.sc
+
+        self.is_v2 = False
+        if _IAMConstants.V2_REST_URL in self.sc.credentials:
+            self.is_v2 = True
+
+    def test_service_stop_start(self):
```

Comment:
Does this test run when the whole suite is run? We don't want by default tests that stop & start the service.

Reply:
I changed the test such that it needs to be run explicitly.